cli/commands/cluster/health_test.go

```diff
@@ -0,0 +1,30 @@
+package cluster
+
+import (
+	"testing"
+
+	"github.com/sensu/sensu-go/cli"
+	client "github.com/sensu/sensu-go/cli/client/testing"
+	test "github.com/sensu/sensu-go/cli/commands/testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHealthCommand(t *testing.T) {
+	assert := assert.New(t)
+
+	cli := newCLI()
+	cmd := HealthCommand(cli)
+
+	assert.NotNil(cmd, "cmd should be returned")
+	assert.NotNil(cmd.RunE, "cmd should be able to be executed")
+	assert.Regexp("health", cmd.Use)
+	assert.Regexp("get sensu health status", cmd.Short)
+}
+
+func newCLI() *cli.SensuCli {
```

Comment:
I think this function is repeated across commands. Perhaps we can move it to the `helpers` package to avoid code duplication?

Reply:
That sounds reasonable, but I'd want to do it in a different PR because refactoring that would touch all the cli tests.